vcd/datasource_vcd_org_vdc_test.go

```diff
@@ -125,9 +125,8 @@ func validateResourceAndDataSource(t *testing.T, configText string, datasourceVd
 	})
 }
 
-func validateDataSource(t *testing.T, configText string, datasourceVdc string) {
+func validateDataSource(t *testing.T, configText string, datasourceVdc string, params StringMap) {
```

Comment:
Maybe I lost some detail but `params` is not used inside this function, is it?

Reply:
yes, removed